src/types/component.rs

```diff
@@ -47,11 +47,44 @@ pub struct InputStream {
 pub struct View {
     pub id: Option<ComponentId>,
     pub children: Option<Vec<Component>>,
+
     pub width: Option<usize>,
     pub height: Option<usize>,
+
+    /// Direction defines how static children are positioned inside the View.
+    /// "row" - Children positioned from left to right.
+    /// "column" - Children positioned from top to bottom.
+    pub direction: Option<ViewDirection>,
+
+    /// Distance between the top edge of this component and the top edge of it's parent. If this
```

Comment:
```suggestion
    /// Distance between the top edge of this component and the top edge of its parent. If this
```
Change `it's` to `its` in other comments too